core/src/com/fallenflame/game/LightInputProcessor.java

```diff
@@ -0,0 +1,52 @@
+package com.fallenflame.game;
+
+import com.badlogic.gdx.InputProcessor;
+import com.fallenflame.game.GameEngine;
+
+import static com.badlogic.gdx.scenes.scene2d.actions.Actions.delay;
+
+public class LightInputProcessor implements InputProcessor {
+   private GameEngine gameEngine;
+
+   public LightInputProcessor(GameEngine g){
+       gameEngine = g;
+   }
+    public boolean keyDown (int keycode) {
+        return false;
+    }
+
+    public boolean keyUp (int keycode) {
+        return false;
+    }
+
+    public boolean keyTyped (char character) {
+        return false;
+    }
+
+    public boolean touchDown (int x, int y, int pointer, int button) {
+        return false;
+    }
+
+    public boolean touchUp (int x, int y, int pointer, int button) {
+        return false;
+    }
+
+    public boolean touchDragged (int x, int y, int pointer) {
+        return false;
+    }
+
+    public boolean mouseMoved (int x, int y) {
+        return false;
+    }
```

Comment:
Why are these all always false?

Reply:
They need to be false so they don't do anything. Basically for these functions, the code defaults to what the LoadingMode InputProcessor does, unless it is the scrolled function. If you look in LoadingMode, the scrolled function returns false. It basically says "we don't use another listener" when those events happen for it to be false.